ui/settings_page.go

```diff
@@ -283,6 +280,23 @@ func (pg *settingsPage) subSectionSwitch(gtx layout.Context, title string, optio
 	})
 }
 
+func (pg *settingsPage) clickableRow(gtx layout.Context, title string, clickable *widget.Clickable, icon *widget.Icon, label decredmaterial.Label) layout.Dimensions {
```

Comment:
If the arguments you'll be passing to a method is becoming too many, you can create a struct type and pass  that instead. For instance, for clickableRow, you can define the method to accept layout.Context and a type "row" as arguments. The type "row" would contain the remaining the parameters you defined on the method.